Makefile

```diff
@@ -1,3 +1,10 @@
+format-python:
+	# Format
+	poetry run black --target-version py38 mlops_utilities/ tests/
+
+lint:
+	poetry run pylint --max-line-length=120 --max-args=8 --output-format=text,pylint_junit.JUnitReporter:lint_result.xml mlops_utilities/
```

Comment:
Configure pylint in `pyproject.toml` instead of using arguments here. The purpose of this file is to configure entire build including linters, formatters, etc. All modern tools support this file as a standard for that. See here - https://pylint.pycqa.org/en/latest/user_guide/usage/run.html - search for `pyproject.toml`

Reply:
This is not resolved, but you can put it into a new separate task